atmel-samd/mpconfigport.h

```diff
@@ -245,5 +247,7 @@ void run_background_tasks(void);
 
 #define CIRCUITPY_AUTORELOAD_DELAY_MS 500
 #define CIRCUITPY_BOOT_OUTPUT_FILE "/boot_out.txt"
+// Scan gamepad every 32ms
+#define CIRCUITPY_GAMEPAD_TICKS 0x1f
```

Comment:
This should be in the express board portion right? Currently this will call ticks but the singleton will always be null.

Reply:
Not sure what you mean. This defines how often the buttons are scanned, and also lets you disable the scanning completely. I put it here, because it's configuration. The `gamepad_singleton` is not related to it, but it won't be always NULL — it's being assigned to in the `gamead_make_new` function.

Reply:
Right, the scanning should be disabled on non-express builds. With this here, non-express boards will still include https://github.com/adafruit/circuitpython/pull/295/files#diff-ae7239759d3115ddffbb5b5fd105f23aR21 and gamepad_ticks().

Reply:
Ah, I see what you mean.